### PR TITLE
small doc clarifications + cleanup

### PR DESCRIPTION
**Pull request by abejgonzalez**
Misc. documentation cleanup and clarifications. I tried to match #235.

### CODE REVIEW

File: docs/Customization/Heterogeneous-SoCs.rst [hunk 1]
```diff
@@ -100,27 +84,20 @@ Named ``MultiRoCCKey``, this key allows you to attach RoCC accelerators based on
...
+Then after that is applied to the parameters, the ``WithMultiRoCCHwacha`` is used to assign to a Hwacha accelerator to a particular ``hartId`` (in this case the ``hartId`` of ``2`` corresponds to the Rocket core).
 Finally, the ``WithMultiRoCC`` mixin is called.
 This mixin sets the ``BuildRoCC`` key to use the ``MultiRoCCKey`` instead of the default.
 This must be used after all the RoCC parameters are set because it needs to override the ``BuildRoCC`` parameter.
 If this is used earlier in the configuration sequence, then MultiRoCC does not work.
 
 This mixin can be changed to put more accelerators on more cores by changing the arguments to cover more ``hartId``'s (i.e. ``WithMultiRoCCHwacha(0,1,3,6,...)``).
+
+
+.. [1] Note, in this section core and tile are used interchangeably but there is subtle distinction between a core and tile (tile contains a core, L1D/I$, PTW).
```
Comment (alonamid) [inline docs/Customization/Heterogeneous-SoCs.rst]: Maybe add quotes around "core" and "tile"?

File: docs/Customization/Heterogeneous-SoCs.rst [hunk 2]
```diff
@@ -100,27 +84,20 @@ Named ``MultiRoCCKey``, this key allows you to attach RoCC accelerators based on
...
-      new freechips.rocketchip.subsystem.WithNBigCores(1) ++
-      new freechips.rocketchip.system.BaseConfig)
+.. literalinclude:: ../../generators/example/src/main/scala/HeteroConfigs.scala
+    :language: scala
+    :start-after: DOC include start: DualBoomAndRocketOneHwacha
+    :end-before: DOC include end: DualBoomAndRocketOneHwacha
 
 In this example, the ``WithRenumberHarts`` relabels the ``hartId``'s of all the BOOM/Rocket cores.
-Then after that is applied to the parameters, the ``WithMultiRoCCHwacha(0)`` is used to assign to ``hartId`` zero a Hwacha (in this case ``hartId`` zero is Rocket).
+Then after that is applied to the parameters, the ``WithMultiRoCCHwacha`` is used to assign to a Hwacha accelerator to a particular ``hartId`` (in this case the ``hartId`` of ``2`` corresponds to the Rocket core).
```
Comment (alonamid) [inline docs/Customization/Heterogeneous-SoCs.rst]: redundant "to"?

Comment (abejgonzalez) [reply docs/Customization/Heterogeneous-SoCs.rst]: Ops.

File: docs/Customization/Heterogeneous-SoCs.rst (hunk 1 above)
Comment (abejgonzalez) [reply docs/Customization/Heterogeneous-SoCs.rst]: Done

**Pull request merged**